### PR TITLE
second commit

### PR DESCRIPTION
**Pull request by hcooch2ch3**

### CODE REVIEW

File: tBvsGR/ViewController.swift [hunk 1]
```diff
@@ -21,6 +21,7 @@ class ViewController: UIViewController {
```
Comment (hcooch2ch3-test) [inline tBvsGR/ViewController.swift:21]: a

File: tBvsGR/ViewController.swift [hunk 2]
```diff
@@ -21,6 +21,7 @@ class ViewController: UIViewController {
     
     @objc func tap() {
```
Comment (hcooch2ch3-test) [inline tBvsGR/ViewController.swift:22]: b

File: tBvsGR/ViewController.swift [hunk 3]
```diff
@@ -21,6 +21,7 @@ class ViewController: UIViewController {
...
         print("tap")
```
Comment (hcooch2ch3-test) [inline tBvsGR/ViewController.swift:23]: c

File: tBvsGR/ViewController.swift [hunk 4]
```diff
@@ -21,6 +21,7 @@ class ViewController: UIViewController {
...
+        print("gesture")
     }
```
Comment (hcooch2ch3-test) [inline tBvsGR/ViewController.swift:25]: d

**Review by hcooch2ch3-test — COMMENTED** [review]
ok

File: tBvsGR/ViewController.swift (hunk 3 above)
Comment (hcooch2ch3-test) [inline tBvsGR/ViewController.swift:23]: hi

File: tBvsGR/ViewController.swift [hunk 5]
```diff
@@ -21,6 +21,7 @@ class ViewController: UIViewController {
...
```
Comment (hcooch2ch3-test) [inline tBvsGR/ViewController.swift:26]: b

File: tBvsGR/ViewController.swift [hunk 6]
```diff
@@ -21,6 +21,7 @@ class ViewController: UIViewController {
...
 
 }
```
Comment (hcooch2ch3-test) [inline tBvsGR/ViewController.swift:27]: c

**Review by hcooch2ch3-test — COMMENTED** [review]
총평

File: tBvsGR/ViewController.swift (hunk 1 above)
Comment (hcooch2ch3-test) [inline tBvsGR/ViewController.swift:21]: as

File: tBvsGR/ViewController.swift (hunk 3 above)
Comment (zdodev) [reply tBvsGR/ViewController.swift:23]: 대박

File: tBvsGR/ViewController.swift (hunk 1 above)
Comment (hcooch2ch3-test) [inline tBvsGR/ViewController.swift:21]: a

File: tBvsGR/ViewController.swift (hunk 2 above)
Comment (hcooch2ch3-test) [inline tBvsGR/ViewController.swift:22]: b

File: tBvsGR/ViewController.swift (hunk 3 above)
Comment (hcooch2ch3-test) [inline tBvsGR/ViewController.swift:23]: x

**Review by hcooch2ch3-test — COMMENTED** [review]
good

**Review by hcooch2ch3-test — COMMENTED** [review]
dd

File: tBvsGR/ViewController.swift (hunk 1 above)
Comment (hcooch2ch3-test) [inline tBvsGR/ViewController.swift:21]: a

**Review by hcooch2ch3-test — COMMENTED** [review]
three

File: tBvsGR/ViewController.swift (hunk 2 above)
Comment (hcooch2ch3-test) [inline tBvsGR/ViewController.swift:22]: sss

**Pull request closed without merging**